delta-app/src/main/java/io/cdap/delta/app/QueueingEventEmitter.java

```diff
@@ -64,8 +64,7 @@ public void emit(DDLEvent event) {
     }
 
     try {
-      eventQueue.put(new Sequenced<>(event, event.getSourceTimestampMillis() == null ? ++sequenceNumber
-        : event.getSourceTimestampMillis()));
+      eventQueue.put(new Sequenced<>(event, ++sequenceNumber));
```

Comment:
Same change required for `emit(DMLEvent event)` method too on line 82

Reply:
good catch :) done :)